ERCS/erc-7673.md

```diff
@@ -0,0 +1,373 @@
+---
+eip: 7673
+title: Distinguishable base256emoji Addresses
+description: Depict Account Addresses As A String of Emoji
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/erc-7673-distinguishable-account-addresses/19461
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-01
+---
+
+## Abstract
+
+Introduce base256emoji for use as the primary input and display for account addresses in all user interfaces.
```

Comment:
It'd be nice to expand a little bit on why you want this, and how your proposal achieves its goal. The abstract should give a high-level but technical overview to the reader.